src/components/FormHtmlEditor.vue

```diff
@@ -0,0 +1,94 @@
+<template>
+  <div class='form-group'>
+    <div :class="classList">
+      <editor
+        v-model="content"
+        :init="editorSettings"
+        v-bind:disabled="disabled"
+        @onFocus="$emit('onFocus')"
+        @onBlur="$emit('onBlur')">
+      </editor>
+    </div>
+    <div v-if="(validator && validator.errorCount) || error" class="invalid-feedback">
+      <div v-for="(error, index) in validator.errors.get(this.name)" :key="index">{{error}}</div>
+      <div v-if="error">{{error}}</div>
+    </div>
+    <small v-if='helper' class='form-text text-muted'>{{helper}}</small>
+  </div>
+</template>
+
+
+<script>
+import { createUniqIdsMixin } from 'vue-uniq-ids'
+import ValidationMixin from './mixins/validation'
+import Editor from '@tinymce/tinymce-vue';
+import 'tinymce/tinymce';
+import 'tinymce/themes/silver';
+import 'tinymce/plugins/link';
+import 'tinymce/plugins/lists';
+
+// Create the mixin
+const uniqIdsMixin = createUniqIdsMixin()
+
+export default {
+  mixins: [uniqIdsMixin, ValidationMixin],
+  components: {
+      Editor
+  },
+  props: [
+    'label',
+    'error',
+    'name',
+    'required',
+    'value',
+    'helper',
+    'disabled',
+    'controlClass'
+  ],
+  computed:{
+    classList(){
+      let classList = {
+        'is-invalid': (this.validator && this.validator.errorCount) || this.error,
+      }
+      if(this.controlClass){
+        classList[this.controlClass] = true
+      }
+      return classList
+    }
+  },
+  data() {
+    return {
+      editorSettings: {
+        inline: true,
+        menubar: false,
+        plugins: [ 'link', 'lists' ],
+        toolbar: "undo redo | link | styleselect | bold italic forecolor | alignleft aligncenter alignright alignjustify | bullist numlist outdent indent",
```

Comment:
Nitpick: Some places you use double quote and in others single quotes. I think it makes sense to use single quotes here as well, to keep things consistent.